server/message_hooks.go

```diff
@@ -609,6 +609,11 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post, c
 			p.API.LogWarn("Error updating the msteams/mattermost post link metadata", "error", err)
 		}
 	}
+
+	if err := p.store.SetUserLastChatSentAt(post.UserId, post.CreateAt*1000); err != nil {
```

Comment:
times are `*1000` because Mattermost uses milliseconds, but we use microseconds in the plugin.